tests/common/macros.rs

```diff
@@ -46,3 +46,38 @@ macro_rules! path_concat {
         pb.to_str().unwrap().to_owned()
     }};
 }
+
+#[macro_export]
+macro_rules! utility_test {
+    () => (
+        fn util_name<'a>() -> &'a str {
+            let split : Vec<&str> = module_path!().split("_").collect();
+            split[1]
```

Comment:
In addition, rather then defining functions, how about using macros? i.e.

```rust
macro_rules! util_name {
  () => ( module_path!().split("_").nth(1).expect("no test name") )
}

macro_rules! new_ucmd {
  () => ( TestScenario::new(util_name!()).ucmd() )
}

macro_rules! at_and_ucmd {
  () => ({
    let ts = TestScenario::new(util_name!());
    (ts.fixtures, ts.ucmd) 
  })
}
```

then in each test module, I can use it like this
```
#[test]
fn test_foo() {
     new_ucmd!()
         .pipe_in(input)
         .succeeds()
         .stdout_only("JBSWY3DPFQQFO33SNRSCC===\n");

     let (at, mut ucmd) = at_and_ucmd!();
     let result = ucmd.arg("-h").run();
     assert_eq!(at.read("file"), result.stdout);
}
```